pkg/apis/discovery/types.go

```diff
@@ -114,8 +114,21 @@ type EndpointConditions struct {
 	// ready indicates that this endpoint is prepared to receive traffic,
 	// according to whatever system is managing the endpoint. A nil value
 	// indicates an unknown state. In most cases consumers should interpret this
-	// unknown state as ready.
+	// unknown state as ready. For compatibility reasons, ready should never be
+	// "true" for terminating endpoints.
 	Ready *bool
+
+	// accepting is identical to ready except it is set regardless of the terminating
+	// state of endpoints. A ready endpoint that is terminating should set this condition
+	// to true.
+	// +optional
+	Accepting *bool
```

Comment:
Some alternative names for this field:
* Serving
* Receiving
* Healthy
* ???

Reply:
I like Serving better than Accepting.  Healthy implies that we are actively health-checking (not yet!)

Open to other ideas...

Reply:
Agree, I like `Serving` here.

Reply:
Thanks, will update the field to `serving`